src/main/java/org/opentripplanner/framework/io/JsonDataListDownloader.java

```diff
@@ -60,17 +60,17 @@ public List<T> download() {
           try {
             return parseJSON(is);
           } catch (IllegalArgumentException e) {
-            log.warn("Error parsing bike rental feed from {}", url, e);
+            LOG.warn("Error parsing bike rental feed from {}", url, e);
           } catch (JsonProcessingException e) {
-            log.warn("Error parsing bike rental feed from {} (bad JSON of some sort)", url, e);
+            LOG.warn("Error parsing bike rental feed from {} (bad JSON of some sort)", url, e);
           } catch (IOException e) {
-            log.warn("Error reading bike rental feed from {}", url, e);
+            LOG.warn("Error reading bike rental feed from {}", url, e);
```

Comment:
Unrelated, but can you replace the words "bike rental"?

Reply:
I've removed the references in https://github.com/opentripplanner/OpenTripPlanner/pull/5795/commits/fd8b26c75945284cf0b8c760965af21977ada59f.